util/src/main/java/io/kubernetes/client/Exec.java

```diff
@@ -269,6 +269,22 @@ protected void handleMessage(int stream, InputStream inStream) throws IOExceptio
               } else super.handleMessage(stream, inStream);
             }
 
+            @Override
+            public void failure(Exception ex) {
+              super.failure(ex);
+              // TODO, it's possible we should suppress this error message, but currently there's
+              // no good place to surface the message, and without it, this will be really hard to
+              // debug.
+              ex.printStackTrace();
+              synchronized (ExecProcess.this) {
+                // Try for a pretty unique error code, so if someone searches they'll find this
+                // code.
+                statusCode = -1975219;
```

Comment:
https://tools.ietf.org/html/rfc6455#section-7.4

```
   3000-3999

      Status codes in the range 3000-3999 are reserved for use by
      libraries, frameworks, and applications.  These status codes are
      registered directly with IANA.  The interpretation of these codes
      is undefined by this protocol.
```
according to the RFC, negative status code doesn't seem recommended.. how about a unique number ranged in 3000-3999?

Reply:
This is a proces status code, not an HTTP/WebSocket status code. I think a process status code can be whatever you want it to be (and negative numbers are definitely allowed.)

Reply:
oh tricked.. yes it's exit code from remote process.